float64 theta_hip
float64 theta_knee
---
float64 x
float64 y